### PR TITLE
allow project hours to drag, modularize project scripts

### PR DESCRIPTION
**Pull request by jsachs**

### CODE REVIEW

File: public/js/projects.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+$("tbody").sortable({
+    items: "> tr",
```
Comment (mmccall) [inline public/js/projects.js]: probably want to adjust this to exclude the totals row.

Comment (jsachs) [inline public/js/projects.js]: Makes sense. Also, I was planning on just limiting it to the Projects page.

**Comment by jsachs** [comment]
@mmccall updated to fix sorting

**Comment by mmccall** [comment]
looks good to me.

**Pull request merged**